### PR TITLE
feat(docs): Document the email suggestion events.

### LINKED ISSUE #2177: new email correction events need to be documented
Follow on from #2009. 

Add the events to https://github.com/mozilla/fxa-content-server/blob/master/docs/client-metrics.md in a format similar to https://github.com/mozilla/fxa-content-server/pull/1806/files#diff-6a80758248a4917855743c82c6daf796

### PR DESCRIPTION
**Pull request by shane-tomlinson**
@vladikoff - can you review these to ensure I have the intent correct?

fixes #2177

### CODE REVIEW

**Comment by vladikoff** [comment]
We also got
```js
'tooltip.' + this.type + '-dismissed';
```
where the default type is "generic" or "mailcheck"

**Comment by vladikoff** [comment]
That "dismissed" is part of tooltip module and not mailcheck

**Comment by shane-tomlinson** [comment]
> 'tooltip.' + this.type + '-dismissed';

OK. We have been using `.` as the separator between the subject and status, so we should probably update these to be consistent.

@kparlante - here are some new events that should make there way to you soon.

**Comment by shane-tomlinson** [comment]
> so we should probably update these to be consistent.

Opened #2193 to make the event names consistent.

**Comment by shane-tomlinson** [comment]
> 'tooltip.' + this.type + '-dismissed';

@vladikoff - added these.

**Comment by vladikoff** [comment]
> so we should probably update these to be consistent.

@shane-tomlinson is this `* tooltip.generic-dismissed` is this inconsistent with the current format?
i.e tooltip is the subject or should it be `tooltip-generic`?

File: docs/client-metrics.md [hunk 1]
```diff
@@ -105,6 +108,9 @@ The event stream is a log of events and the time they occurred while the user is
 #### settings/avatar/gravatar
 #### signin
 #### signup
+* tooltip.mailcheck-suggested - an email address correction was suggested
+* tooltip.mailcheck-used - an email address correction was chosen by the user
+* tooltip.mailcheck-dismissed - an email address correction tooltip was dismissed without the selection being made.
```
Comment (vladikoff) [inline docs/client-metrics.md]: :+1:

**Pull request merged**